sr_object_tracking/launch/tracking.launch

```diff
@@ -15,9 +15,7 @@
   </group>
 
   <group unless="$(arg kinect)">
-    <node pkg="uvc_camera" type="uvc_camera_node" name="uvc_camera" output="screen">
-        <remap from="/image_raw" to="/camera/rgb/image_color"/>
-    </node>
+    <include file="$(find autopic)/launch/camera.launch" />
```

Comment:
No! You can't have this public launch file dependent on a private project :)

The way to do it is to add a no camera option to this launch file and then call it from autopic.